notebooks/SMAP/3.0 SMAP Quality Flags.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<p>As with the quality flags, I suggest we get these from the Dataset attributes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>